webpack.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   output: {
     path: path.join(__dirname, 'jekyll/assets/js'),
     publicPath: '',
-    filename: 'assets/js/dist/[name]-[contenthash].min.js'
```

Comment:
just a note, if we find it too difficult to configure hashing with jekyll plugins, we still might want to use this bit. But that's an aside, just in case, for later.